src/main/kotlin/fi/metatavu/timebank/api/controllers/DailyEntryController.kt

```diff
@@ -87,21 +92,25 @@ class DailyEntryController {
         var internalTime = 0
         var projectTime = 0
         var date = LocalDate.now()
+        lateinit var worktimeCalendarId: UUID
```

Comment:
I'd change this so that you first find the worktimeCalendar id from the entries (there should be only one since we are going through entries of a single day + you might wanna throw error if that isn't the case) and assign that found ID to this variable instead of using lateinit. This cleans the code a bit and in the current solution you are assigning the variables multiple times since you are looping through the entries.